src/main/java/net/sf/rails/ui/swing/GameUIManager.java

```diff
@@ -182,6 +182,15 @@ public void init(RailsRoot root, boolean wasLoaded, SplashWindow splashWindow) {
         for (Player player : getPlayers()) {
             currentGuiPlayerNames.add(player.getId());
         }
+
```

Comment:
What does `System.getProperty("local.player.name")` contain and where is it set?

Reply:
I believe it's a fallback allowing a user to set their player name via a system property when starting the game vs configuration. This logic existed previous, not sure if it's value anymore but since I didn't write it and don't know the use case I'm not comfortable removing it